lib/steps/delete_parent_branch_step.go

```diff
@@ -23,7 +23,15 @@ func (step DeleteParentBranchStep) CreateUndoStep() Step {
 	}
 }
 
+func (step DeleteParentBranchStep) GetAutomaticAbortErrorMessage() string {
```

Comment:
Hmm, there must be a better way to do this that avoids all this duplication. Traditionally I would have moved this method into the base class and then override in subclasses as needed. Go achieves the same thing with composition. [Here](http://stackoverflow.com/questions/31997753/golang-reuse-a-function-across-multiple-structs-to-satisfy-an-interface) is an idea on how this could be done.